File: kildey004/programacaobasica/Questao11.java
import java.util.Scanner;

public class Questao11 { 
    public static void main (String[] args) {
        double n1;
        double p1;
        double n2;
        double p2;
        double n3;
        double p3;
        double n4;
        double p4;
        double tn;
        double tp;
        double media;
        Scanner s;

        s = new Scanner(System.in);
        System.out.println("Primeira nota:");
        n1 = s.nextDouble();
        System.out.println("peso da nota1:");
        p1 = s.nextDouble();
        System.out.println("Segunda nota:");
        n2 = s.nextDouble();
        System.out.println("peso da nota2:");
        p2 = s.nextDouble();
        System.out.println("Terceira nota:");
        n3 = s.nextDouble();
        System.out.println("peso da nota3:");
        p3 = s.nextDouble();
        System.out.println("Quarta nota:");
        n4 = s.nextDouble();
        System.out.println("peso da nota4:");
        p4 = s.nextDouble();

        tn = n1 * p1 + n2 * p2 + n3 * p3 + n4 * p4;
        tp = p1 + p2 + p3 + p4;
        media = tn / tp;

        System.out.println(" A media da sua nota eh: " + media);
    }
}

File: kildey004/programacaobasica/questao11.py
print("**Calculador de media ponderada**")

n1 = float(input("Digite a primeira nota: "))
p1 = float(input("Digite o peso da primeira nota: "))
n2 = float(input("Digite a segunda nota: "))
p2 = float(input("Digite o peso da segunda nota: "))
n3 = float(input("Digite a terceira nota: "))
p3 = float(input("Digite o peso da terceira nota: "))
n4 = float(input("Digite a quarta nota: "))
p4 = float(input("Digite o peso da quarta nota: "))


tn = n1 * p1 + n2 * p2 + n3 * p3 + n4 * p4

tp = p1 + p2 + p3 + p4

mediap = tn / tp

print("A media da sua nota eh: " + str(mediap))

